src/dodal/beamlines/i22.py

```diff
@@ -198,6 +198,7 @@ def undulator(
         bl_prefix=False,
         poles=80,
         length=2.0,
+        id_gap_lookup_table_path="/dls_sw/i22/software/daq_configuration/lookup/BeamLine_Undulator_toGap.txt",
```

Comment:
Are all these files in the same format?

Reply:
As in both file extension and internal format

Reply:
This will require making blueapi mount at least `/dls_sw/i22/software/daq_configuration/lookup/` or move that file into a Kubernetes ConfigMap and mounting it- how complex is the file and how often is it changing?

Reply:
the one for i22 is empty. the one for i03 was made using harmonics 3 5 7 and the date indicated inside is 2022

Reply:
I think the Undulator device won't be used in bluesky anytime soon. I am not familiar with how alignment works there at the moment, and in the near/mid term future it will use the new redis based lookup tables. 

for this PR I suggest we just leave it as-is, I can watch at i22 when could we change this.

@DiamondJoseph can we resolve this thread?